docs/excel/excel-add-ins-pivottables.md

```diff
@@ -211,6 +211,37 @@ The following examples use the outline and tabular styles, respectively. The cod
 
 ![A PivotTable using the tabular layout.](../images/excel-pivots-tabular-layout.png)
 
+### Other PivotLayout functions
+
+By default, PivotTables adjust row and column sizes as needed. This is done when the PivotTable is refreshed. `PivotLayout.autoFormat` specifies that behavior. Any row or column size changes made by your add-in persist when `autoFormat` is `false`. Additionally, the default settings of a PivotTable keep any custom formatting in the PivotTable (such as fills and font changes). Set `PivotLayout.preserveFormatting` to `false` to apply the default format when refreshed.
```

Comment:
Out of curiosity, why didn't you include a link to [autoFormat](https://docs.microsoft.com/en-us/javascript/api/excel/excel.pivotlayout?view=excel-js-preview#autoFormat)?

Reply:
Mostly because the anchor links in the ref docs are going to change soon.